File: trainning1team/src/main/java/com/bnksystem/trainning1team/dto/Equip/RentalStatusResponse.java
package com.bnksystem.trainning1team.dto.Equip;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
public class RentalStatusResponse {
    private Long id;
    private String serialNo;
    private String equipmentName;
    private String statusName;
    private String employeeNo;
    private LocalDateTime regAt;
}

File: GearTracker/app/src/main/java/com/bsys/geartracker/data/model/response/RentalStatusResponse.kt
package com.bsys.geartracker.data.model.response

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime

data class RentalStatusResponse (
    @SerializedName("id") val id: Long,
    @SerializedName("serialNo") val serialNo: String,
    @SerializedName("equipmentName") val equipmentName: String,
    @SerializedName("statusName") val statusName: String,
    @SerializedName("employeeNo") val employeeNo: String,
)